.circleci/config.yml

```diff
@@ -22,7 +22,11 @@ jobs:
       - run:
           command: |
             ./run ./configure.py --compiler << parameters.compiler >> --c++-standard << parameters.standard >>
-            ./run ninja -C build/<< parameters.mode >>
+            if [ << parameters.standard >> = "dev" ]; then
+              ./run ninja -C build/<< parameters.mode >> all checkheaders
```

Comment:
I think you have a typo in the commit message: you said that checkheaders "is not excluded from all", I think you meant to say either it is not **included** in all, or **excluded** (without the not) from all?

Reply:
ahh, true. will correct the commit message.